keel-sql/src/main/resources/db/changelog/20201006-add-unhealthy-repository.yml

```diff
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-unhealthy-repo
+      author: emjburns
+      changes:
+        - createTable:
+            tableName: unhealthy
+            columns:
+              - column:
+                  name: resource_id
+                  type: varchar(255)
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: time_detected
+                  type: bigint
```

Comment:
Please can we use `timestamp(3)` like everything else does?

Reply:
Fine Rob. Fine. Rob.